src/HappoRunner.js

```diff
@@ -170,6 +170,15 @@ window.happo = {
             width,
           } = getFullRect(rootNodes);
 
+          const overlay = document.createElement('div');
```

Comment:
It might be nice to have a comment here explaining why we are creating an overlay. Or perhaps a more descriptive variable name than `overlay`. Maybe `screenshotOverlay` or `screenshotBox`?